images/manageiq-base/Dockerfile

```diff
@@ -9,10 +9,13 @@ ARG MIQ_ORG=ManageIQ
 ARG SUI_ORG=ManageIQ
 ARG APPLIANCE_ORG=ManageIQ
 
+ARG ARCH=x86_64
+
 ENV TERM=xterm \
     CONTAINER=true \
     APP_ROOT=/var/www/miq/vmdb \
     APPLIANCE_ROOT=/opt/manageiq/manageiq-appliance \
+    APPLIANCE_PG_DATA=/var/lib/pgsql/data \
```

Comment:
Why was this added?

Reply:
this was needed because of the difference in the way in which we were installing postgresql on ppc64le as compared to Intel, now that we have consistency there, this can be removed, I will test